buildSrc/README.md

```diff
@@ -1,5 +1,8 @@
 ## Build Source
 
+> [!NOTE]
```

Comment:
I would argue otherwise, as contributions could be to `buildSrc` as well. Additionally, It's a great way to organize our documentation in similar places. We could have it under a separate category that articulates its alignment with buildSrc and such. wdyt?